ui/app/styles/core/forms.scss

```diff
@@ -109,11 +109,16 @@ label {
 .input.variable {
   font-family: $family-monospace;
 }
+
+.select select[disabled],
 .input[disabled],
 .textarea[disabled] {
   border-color: $grey-light;
-  background-color: $white-ter;
-  color: $grey-light;
+  background-color: $ui-gray-100;
+  color: $ui-gray-500;
+  &:hover {
```

Comment:
this was being overridden somewhere and I couldn't for the life of me (after way too much time) figure out where 🤯 so explicitly defined keeping the border if the input is hovered while disabled. Use this in favor of adding `!important` to line 116, because I remember a while ago being told we were trying to avoid using that